src/View/mainpage/DrawROIWindow/Drawing.py

```diff
@@ -115,15 +117,44 @@ def _display_pixel_color(self):
                 colors = QColor(c).getRgbF()
                 self.according_color_dict[(x_coord, y_coord)] = colors
 
-            color = QtGui.QColor()
-            color.setRgb(90, 250, 175, 200)
-            points = get_pixel_coords(self.according_color_dict, self.rows,
-                                      self.cols)
-            for x_coord, y_coord in points:
-                self.q_image.setPixelColor(x_coord, y_coord, color)
+            points = get_pixel_coords(self.according_color_dict, self.rows, self.cols)
+
+            self._set_color_of_pixels(points)
```

Comment:
very very picky, no need to change at all. But try to group together common functionality rather than separating each on a new line.

Reply:
fixed